modin/experimental/cloud/rayscale.py

```diff
@@ -158,17 +160,108 @@ def _conda_requirements(self):
         return reqs_with_quotes
 
     def _update_conda_requirements(self, setup_commands: str):
-        return setup_commands.replace(
+        setup_commands = setup_commands.replace(
             "{{CONDA_PACKAGES}}", " ".join(self._conda_requirements())
         )
 
+        return setup_commands.replace(
+            "{{INSTALL_MODIN_COMMAND}}", self._get_modin_install_command()
+        )
+
     @staticmethod
     def _get_python_version():
         major = sys.version_info.major
         minor = sys.version_info.minor
         micro = sys.version_info.micro
         return [f"python>={major}.{minor}", f"python<={major}.{minor}.{micro}"]
 
+    @staticmethod
+    def _git_state():
```

Comment:
First of all, this approach is dangerous: https://stackoverflow.com/questions/3878624/how-do-i-programmatically-determine-if-there-are-uncommitted-changes/3879077#3879077
(long story short - do not parse stuff intended for humans, always try to use machine-readable output)

Second, I'm not entirely sure we need all this functionality - it's pretty easy to edit `ray-autoscaler.yml` manually during developmentto make it install modin from source, and I highly doubt any of our users would need this funcitonality (now Occam's Razor comes into play).

Let's not overcomplicate things.

Reply:
Make sense!